modules/core/src/main/java/org/apache/fluo/core/client/FluoAdminImpl.java

```diff
@@ -516,6 +517,23 @@ public boolean oracleExists() {
     return oracleExists(getAppCurator());
   }
 
+  public static int numOracles(CuratorFramework curator) {
```

Comment:
Seems like this method is only called in one place and could be inlined.

Reply:
Done